mgr.go

```diff
@@ -398,15 +397,19 @@ func newSysboxMgr(ctx *cli.Context) (*SysboxMgr, error) {
 
 func (mgr *SysboxMgr) Start() error {
 
-	// setup rootfs watcher (to detect container removal)
-	w, err := fsnotify.NewWatcher()
+	// setup the container rootfs monitor (detects container removal)
+	cfg := &fileMonitor.Cfg{
+		EventBufSize: 10,
```

Comment:
Initially I had a higher delay (100ms), but some Sysbox tests were failing when doing 

```
docker stop <container>
check /var/lib/sysbox/docker is empty
```

This is why I reduced the delay. We could increase it, but it will require that I modify the tests to add a delay between stopping the container and checking sysbox has cleaned up after it.

I don't see much problem with the 1ms interval either, the goroutine that runs every interval is pretty tiny so add little overhead.

Reply:
Hi @rodnymolina, I've adjust the polling period in the rootfs monitor from 1ms -> 50ms. Let me know if this looks good. Thanks!